back/config/schemas/settings.schema.json.erb

```diff
@@ -1226,6 +1226,18 @@
           "allowed": { "type": "boolean", "default": true },
           "enabled": { "type": "boolean", "default": false }
         }
+      },
+
+      "analysis": {
+        "type": "object",
+        "title": "AI-powered input analysis",
+        "description": "Enables the ability to create an AI-powered analysis for inputs on the platform.",
+        "additionalProperties": false,
+        "required": ["allowed", "enabled"],
+        "properties": {
+          "allowed": { "type": "boolean", "default": true },
+          "enabled": { "type": "boolean", "default": false }
+        }
```

Comment:
I've added the feature flag in the BE branch as well, but in [a different (better) way.](https://github.com/CitizenLabDotCo/citizenlab/pull/5428/files#diff-da22c1e566805b9894f113de572c72ce58c1a9410ebdebc13357fa603eeb6b07) For engines, we specify the features in code, which in the end has the same effect of generating this same kind of objects in the settings schema.

So best to take this out before merging, or merge both branches together first.